data/alpha/register/social-housing-provider-legal-entity.yaml

```diff
@@ -0,0 +1,9 @@
+fields:
+- social-housing-provider-legal-entity
```

Comment:
yeah, sub register (signifies if the entry is a non-profit, profit or local authority)